chain/manager.go

```diff
@@ -1091,25 +1087,13 @@ func (m *Manager) updateV2TransactionProofs(txns []types.V2Transaction, from, to
 		return nil, fmt.Errorf("reorg path from %v to %v is too long (-%v +%v)", from, to, len(revert), len(apply))
 	}
 
-	var index types.ChainIndex
-	defer func() {
-		if panicErr := recover(); panicErr != nil {
```

Comment:
I know we fixed a couple of spots now that avoid panicking, including this one, but should we get rid of the recover here? Feels like it doesn't hurt to keep it around for a bit.